### ISSUE TITLE
move off jcenter and put artifacts on maven central

### ISSUE THREAD

**Issue by kenyee** [ThomasGorisse/sceneform-android-sdk#27]
jcenter is going away...would be nice if this were on maven central

**Comment by RGregat** [comment]
I thought the same the last days. Currently I'm using this lib directly without any Online-Repository.

**Comment by ThomasGorisse** [comment]
Maven central is (from what I have seen) a nightmare to integrate. If you have some knowledge or good link about it, I'm interesting.
Waiting for that, there is still the Jitpack solution already in place : [https://thomasgorisse.github.io/sceneform-android-sdk/dependencies](https://thomasgorisse.github.io/sceneform-android-sdk/dependencies)

**Comment by kenyee** [comment]
Marton wrote a decent article about the process...it's not too bad once you set up everything.
https://proandroiddev.com/publishing-android-libraries-to-mavencentral-in-2021-8ac9975c3e52

**Comment by ThomasGorisse** [comment]
I confirm that it's a headache not only for configuring it but also for deploying a new release.
JitPack is so so so much easier fro everything and also able the snapshot usage.

**Comment by ThomasGorisse** [comment]
Anyone has an idea of what will be the future standard maven repository used by default by AndroidStudio default gradle file since jCenter is dying ?

**Comment by suryanadiminti99** [comment]
> Anyone has an idea of what will be the future standard maven repository used by default by AndroidStudio default gradle file since jCenter is dying ?

I supported JitPack. It is very easy for releasees.

**Comment by ThomasGorisse** [comment]
Sceneform is now available on MavenCentral.
Fixed : 2494f6eeae57cc610ecb8bea01812754943f61d3

**Comment by ThomasGorisse** [comment]
Can you @suryanadiminti99, @kenyee and @RGregat please test the mavenCentral() integration clearing the gradle cache and removing jCenter/jitpack.io with this?
_/build.gradle_
```gradle
allprojects {
    repositories {
        ...
        mavenCentral()
    }
}
```

_app/build.gradle_
```gradle
dependencies {
     implementation("com.gorisse.thomas.sceneform:sceneform:1.18.8")
}
```

**Comment by ThomasGorisse** [comment]
Fixed 2494f6eeae57cc610ecb8bea01812754943f61d3

**Comment by RGregat** [comment]
Sorry for my absence, I will try it in the next few days!
